src/main/java/com/autotune/database/helper/DBHelpers.java

```diff
@@ -172,6 +181,46 @@ public static KruizeRecommendationEntry convertKruizeObjectTORecommendation(Krui
                 }
                 return kruizeRecommendationEntry;
             }
+
+            /**
+             *
+             * @param kruizeResultsEntry
+             * @param mainKruizeExperimentMap
+             * This method fetches the result data from DB and adds it into the ExperimentResultData object.
+             */
+            public static void convertDBResultsDataToLocalStorage(KruizeResultsEntry kruizeResultsEntry, Map<String, KruizeObject> mainKruizeExperimentMap) {
+                ExperimentResultData experimentResultData = new ExperimentResultData();
+                try (Session session = KruizeHibernateUtil.getSessionFactory().openSession()) {
+                    String hql = "FROM KruizeResultsEntry WHERE experimentName = :experimentName " +
+                            "AND intervalStartTime = :intervalStartTime " +
+                            "AND intervalEndTime = :intervalEndTime";
```

Comment:
Please use same template as what @msvinaykumar has for the createExperiment

Reply:
Updated